Library/ViewModels/RewardCellViewModel.swift

```diff
@@ -201,6 +203,14 @@ RewardCellViewModelOutputs {
         : Strings.Select_this_reward()
     }
 
+    self.shippingLocationsStackViewHidden = reward.map {
+      $0.shipping.summary == nil
+    }
+
+    self.shippingLocationsSummaryLabelText = reward.map {
+      " " + ($0.shipping.summary ?? "")
```

Comment:
Instead of adding a space here you can set up the spacing something like this `|> UIStackView.lens.spacing .~ Styles.gridHalf(1)` in the `RewardCell`.

Reply:
Thank you, @cdolm92 . I will do that.